packages/kbn-interpreter/src/server/server_registries.js

```diff
@@ -60,11 +58,7 @@ export const populateServerRegistries = types => {
         require(path);
       });
 
-      Object.keys(global).forEach(key => {
-        if (!globalKeys.includes(key)) {
-          delete global[key];
-        }
-      });
+      delete global.canvas;
```

Comment:
We add the global for plugins to use. They all use `canvas.register`, so the global needs to exist when the code is run (ie. when `require(path)` is run).

My understanding is that it's removed to avoid complaints from our test runners, which check for added globals in the node runtime and fail if any are unexpectedly added.